helm_deploy/templates/deployment.yaml

```diff
@@ -126,6 +126,21 @@ spec:
               value: {{ .Values.variables.appStore }}
             - name: ENV
               value: {{ .Values.variables.environment}}
+            - name: ENTRA_CLIENT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: portal-sp-certificate
+                  key: entra-client-id
+            - name: ENTRA_TENANT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: portal-sp-certificate
+                  key: entra-tenant-id
+            - name: ENTRA_CERTIFICATE_DATA
+              valueFrom:
+                secretKeyRef:
+                  name: portal-sp-certificate
+                  key: entra-certificate-data
             - name: LAA_PORTAL_SP_CERT
```

Comment:
Assuming this secret name is going to change once we're ready to move to other envs

Reply:
Yeah I didn't rename that did I, might as well do that today as well